app/services/telegram/incoming_message_service.rb

```diff
@@ -81,6 +81,7 @@ def additional_attributes
       # TODO: Remove this once we show the social_telegram_user_name in the UI instead of the username
       username: telegram_params_username,
       language_code: telegram_params_language_code,
+      social_profiles: { telegram: telegram_params_username },
       social_telegram_user_id: telegram_params_from_id,
       social_telegram_user_name: telegram_params_username
```

Comment:
We standardize the external channel user_name/id by appending "social_" in the additional attributes. We will deprecate the attribute "social_profiles" soon. Let's use social_telegram_user_name here.

[Reference](https://github.com/chatwoot/chatwoot/pull/8802)